docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,51 @@
+# Cluster certificate rotation
+
+Instructions on how to rotate TLS CA and certs for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in healthy state before rotating the CA and certs.
+- The `_output` directory for your cluster deployment, including working SSH key for accessing the master nodes and `kubeconfig`.
+
+## Preparation
+
+**CAUTION**: Before rotation, it's preferrable to back up your cluster:
+
+1. On an etcd node, take a backup of the current state.
+2. //TODO recommend etcd backup tool, extract the existing state from etcd. Copy this back to your working machine as a precaution, in case the control plane goes down.
+3. Keep a copy of your current _output directory
+
+## Rotation
+
+**CAUTION**: Rotating certificates will cause cluster downtime
+
+run `aks-engine rotate-certs`
+
+`aks-engine rotate-certs` will:
+
+- Use part of the AKS-Engine deploy code to generate the certificates we need, using the same cert properties and default expiration as new cluster certs
+- From master rotate etcd CA and certificates in all masters
+- From master rotate CA and certs in the nodes
```

Comment:
```suggestion

```

Reply:
Unneeded?

Reply:
there's a missing word. I think it should be apiserver certificates.